### PR TITLE
refactor(core): Rename `ActiveWorkflowRunner` to `ActiveWorkflowManager` (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
We have `WorkflowService`, `ExecutionService`, `WorkflowExecutionService`, `WorkflowRunner`, `ActiveExecutions`, `ActiveWorkflows`, and `ActiveWorkflowRunner` - the overlap in semantics and responsibilities makes the workflow execution lifecycle hard to follow.

This PR is the first of a series:
- Rename `ActiveWorkflowRunner` to `ActiveWorkflowManager`
- Combine `WorkflowExecutionService` with `WorkflowRunner` as `RunnerService`
- Move execution stopping logic from `ExecutionService` to `RunnerService`

### CODE REVIEW

File: packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts [hunk 1]
```diff
@@ -179,7 +179,7 @@ export = {
 			await replaceInvalidCredentials(updateData);
 			addNodeIds(updateData);
 
-			const workflowRunner = Container.get(ActiveWorkflowRunner);
+			const workflowRunner = Container.get(ActiveWorkflowManager);
```
Comment (despairblue) [inline packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts:182]:
```suggestion
			const workflowManager = Container.get(ActiveWorkflowManager);
```

Should we rename this as well? Or would that make the PR too big?

File: packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts [hunk 2]
```diff
@@ -268,7 +268,7 @@ export = {
 				return res.status(404).json({ message: 'Not Found' });
 			}
 
-			const workflowRunner = Container.get(ActiveWorkflowRunner);
+			const workflowRunner = Container.get(ActiveWorkflowManager);
```
Comment (despairblue) [inline packages/cli/src/PublicApi/v1/handlers/workflows/workflows.handler.ts:271]:
```suggestion
			const workflowManager = Container.get(ActiveWorkflowManager);
```

File: packages/cli/test/integration/active-workflow-manager.test.ts [hunk 3]
```diff
@@ -47,15 +47,15 @@ Object.assign(loader.loadedNodes, {
...
+let runner: ActiveWorkflowManager;
 
 let createActiveWorkflow: () => Promise<WorkflowEntity>;
 let createInactiveWorkflow: () => Promise<WorkflowEntity>;
 
 beforeAll(async () => {
 	await testDb.init();
 
-	runner = Container.get(ActiveWorkflowRunner);
+	runner = Container.get(ActiveWorkflowManager);
```
Comment (despairblue) [inline packages/cli/test/integration/active-workflow-manager.test.ts:58]:
```suggestion
	manager = Container.get(ActiveWorkflowManager);
```

Maybe 🤔

**Review by despairblue — CHANGES_REQUESTED** [review]
I checked how much conflicts this would cause with feature/rbac, but it's manageable. 

I found some variables that still mention `Runner`. I saw that you renamed most of the others to be called `...Manager`, maybe we can rename the remaining ones as well to have this consistent. What do you think?

**Comment by ivov** [comment]
Thanks for reviewing!

> how much conflicts 

Happy to wait! I'll leave it up to you.

> some variables that still mention

Fixed!

**Review by despairblue — DISMISSED** [review]
> Happy to wait! I'll leave it up to you.

No need to wait. The merge should be easy enough.

**Review by despairblue — APPROVED** [review]
